lib/Doctrine/DBAL/Query/Expression/CompositeExpression.php

```diff
@@ -69,9 +69,9 @@ public function addMultiple(array $parts = [])
      *
      * @param mixed $part
```

Comment:
This is probably `self|string`, the same as the element of `addMultiple($parts)`.